c2corg_ui/templates/image/view.html

```diff
@@ -33,6 +32,10 @@
 </%block>
 
 <%block name="moduleConstantsValues">
+  <%
+      geometry = loads(image['geometry']['geom'])['coordinates'] if image['geometry'] and image['geometry']['geom'] else Null
```

Comment:
What about Null/None?

``if image['geometry'] and image['geometry']['geom']`` => OK for me, but perhaps more consistent to use the same syntax for testing as below: ``if image.get('geometry') and image.get('geometry').get('geom')``?

Anyway I don't find it very consistent to get the coordinates this way. It would be better to use the same way than for waypoint for instance:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/view.html#L52
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/views/waypoint.py#L48

So you probably have to add ``'geometry': self._get_geometry(waypoint['geometry']['geom']),`` to https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/views/image.py#L42
Ditto for the archive view, a few lines below.